blockchain/pool.go

```diff
@@ -142,8 +140,9 @@ func (pool *BlockPool) IsCaughtUp() bool {
 		maxPeerHeight = MaxInt(maxPeerHeight, peer.height)
 	}
 
-	isCaughtUp := (height > 0 || time.Now().Sub(pool.startTime) > 5*time.Second) && (maxPeerHeight == 0 || height >= maxPeerHeight)
-	pool.Logger.Info(Fmt("IsCaughtUp: %v", isCaughtUp), "height", height, "maxPeerHeight", maxPeerHeight)
+	// ('receive at least one block' or 'wait at least 5 seconds') and ('all peers are also at 0' or 'my height is higher than the highest peer I know')
```

Comment:
Beforehand it was extracted like this ```height := pool.height``` and then height was used everywhere. Personally I think it is more readable to inline these short statements, as it means that you then don't have to refer back to the local variable to understand the meaning.